source/documentation/archive/serving_data/fuseki1.md

```diff
@@ -0,0 +1,450 @@
+Initially, the dataset is empty.
+
+    ## ---------------------------------------------------------------
+    ## Updatable in-memory dataset.
+
+    <#service1> rdf:type fuseki:Service ;
+        fuseki:name                       "ds" ;       # http://host:port/ds
+        fuseki:serviceQuery               "query" ;    # SPARQL query service
+        fuseki:serviceQuery               "sparql" ;   # SPARQL query service
+        fuseki:serviceUpdate              "update" ;   # SPARQL query service
+        fuseki:serviceUpload              "upload" ;   # Non-SPARQL upload service
+        fuseki:serviceReadWriteGraphStore "data" ;     # SPARQL Graph store protocol (read and write)
+        # A separate read-only graph store endpoint:
+        fuseki:serviceReadGraphStore      "get" ;      # SPARQL Graph store protocol (read only)
+        fuseki:dataset                   <#dataset-mem> ;
+        .
+
+    <#dataset-mem> rdf:type ja:RDFDataset .
+
+### Service 2
+
+This service offers a number of endpoints. It is read-only, because
+only read-only endpoints are defined (SPARQL Query and HTTP GET
+SPARQl Graph Store protocol). The dataset is a single in-memory
+graph:
+
+This service offers read-only access to a dataset with a single
+graph of data.
+
+    <#service2> rdf:type fuseki:Service ;
+        fuseki:name                     "books" ;    # http://host:port/books
+        fuseki:serviceQuery             "query" ;    # SPARQL query service
+        fuseki:serviceReadGraphStore    "data" ;     # SPARQL Graph store protocol (read only)
+        fuseki:dataset           <#books> ;
+        .
+
+    <#books>    rdf:type ja:RDFDataset ;
+        rdfs:label "Books" ;
+        ja:defaultGraph
+          [ rdfs:label "books.ttl" ;
+            a ja:MemoryModel ;
+            ja:content [ja:externalContent <file:Data/books.ttl> ] ;
+          ] ;
+        .
+
+### Service 3
+
+This service offers SPARQL query access only to a TDB database. The
+TDB database can have specific features set, such as query timeout
+or making the default graph the union of all named graphs.
+
+    <#service3>  rdf:type fuseki:Service ;
+        fuseki:name              "tdb" ;       # http://host:port/tdb
+        fuseki:serviceQuery      "sparql" ;    # SPARQL query service
+        fuseki:dataset           <#dataset> ;
+        .
+
+    <#dataset> rdf:type      tdb:DatasetTDB ;
+        tdb:location "DB" ;
+        # Query timeout on this dataset (1s, 1000 milliseconds)
+        ja:context [ ja:cxtName "arq:queryTimeout" ;  ja:cxtValue "1000" ] ;
+        # Make the default graph be the union of all named graphs.
+        ## tdb:unionDefaultGraph true ;
+         .
+
+## SPARQL Over HTTP
+
+**SOH** (SPARQL Over HTTP) is a set of command-line scripts for
+working with SPARQL 1.1. SOH is server-independent and will work
+with any compliant SPARQL 1.1 system offering HTTP access.
+
+See the [SPARQL Over HTTP](soh.html) page.
```

Comment:
`/documentation/fuseki2/soh.html`?

Reply:
Sorry this got mixed together.

I've reworked the archiving commit and fixed the links so that work in the archived area.

I don't think we should promise long term that archived versions will always work. 

They probably shouldn't be findable by navigating the website.

Reply:
Makes sense :+1: